Capabilities pane - disable authentication

micgibso opened the pull request:
Signed-off-by: micgibso <MICGIBSO@uk.ibm.com>

## What type of PR is this ? 

- [ ] Bug fix
- [ X] Enhancement

## What does this PR do ?
Describes the new Capabilities pane function to disable metrics.

## Which issue(s) does this PR fix ?
https://github.com/eclipse/codewind/issues/2712

## Does this PR require a documentation change ?
Yes

## Any special notes for your reviewer ?
Topic checked with Acrolinx.

micgibso commented:
@markcor11 Mark, here's the PR for disabling metrics, please review, thx.

markcor11 commented on docs/_documentations/performance.md:
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
+- **Project status:** Provides current project status.
+- **Run Load Feature:** Provides information about whether load pressure can be applied to your project.
+- **Live monitoring:** Use this information to determine whether you can view live metrics. For example, Appsody projects protect the metrics endpoint by requiring authentication. Authentication can stop the performance dashboard from accessing the endpoint and from being able to retrieve performance metrics. You can programmatically resolve disabling authentication by following the suggestions t ...
+- **Benchmarks:** Benchmark your project as you run your tests. The charts display comparisons across load runs.
```
Exactly,   Benchmarks are a collection of completed load test results displayed on a chart and in a table.

micgibso replied:
Changed to your suggestion.

micgibso commented on docs/_documentations/performance.md:
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
-![performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+![Performance dashboard](images/performanceguide/performancedash_window.png){:height="375px" width="800px"}<br>
+
+## Viewing the status of your project
```
Amended.

micgibso commented on docs/_documentations/performance.md:
```diff
@@ -28,7 +28,20 @@ You will learn how to add and enable monitoring to your application. Then, you c
+
+- **Project status:** Provides current project status.
+- **Run Load Feature:** Provides information about whether load pressure can be applied to your project.
+- **Live monitoring:** Use this information to determine whether you can view live metrics. For example, Appsody projects protect the metrics endpoint by requiring authentication. Authentication can stop the performance dashboard from accessing the endpoint and from being able to retrieve performance metrics. You can programmatically resolve disabling authentication by following the suggestions t ...
```
Redrafted, but please recheck.

micgibso commented:
@markcor11 Review comments incorporated but please recheck. Appreciate the graphic quality isn't very good/out of date, could you grab an updated screen shot and paste it in here? I'll then update.  Thank you.

micgibso commented:
I've renamed the screen `Project Capabilities Overview` and updated it all accordingly. Considered putting the icon switch at the top but the first thing you want to know about is the content of the screen so left it at the bottom of the section.  Comments appreciated.

micgibso commented:
@sishida Please add approving review. thx.
